components/Hero.tsx

```diff
@@ -1,27 +1,157 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import styled from "@emotion/styled";
+import Image from "next/image";
+import { useQueryClient } from "react-query";
 import { useRouter } from "next/router";
+import { matchSorter } from "match-sorter";
```

Comment:
just looked into this, looks pretty good

Reply:
Huge time saver indeed